backoffice/src/Models/FormModels/LetterForm.cs

```diff
@@ -11,6 +12,12 @@ public LetterForm()
     }
 
 
+    public LetterForm(Guid droitId)
+    {
+        DroitId = droitId;
+    }
+
+
     public LetterForm(Letter letter)
     {
         LetterId = letter.Id;
```

Comment:
I wonder if there's something smart we can do about getting default Subjects for each of the types. Maybe in here, or in the controller. 

Probably some sort of switch, or having another property in the enum